gpao/project.py

```diff
@@ -18,17 +18,16 @@ def handler(obj):
 
 class Project:
 
-    """ Project class """
+    """ Project classe """
 
-    _cpt_id = 0
+    cpt_id = 0
+    reorganized = False
 
     def __init__(self, name, jobs=None, deps=None):
         self.name = name
 
         self.jobs = []
-
-        if jobs is not None:
-            self.jobs.extend(jobs)
+        self.jobs.extend(jobs)
```

Comment:
pourquoi tu enleves le `if jobs is not None:` ?
ça va plus marcher avec le paramètre par défaut (ou sinon remplace `jobs=None` par jobs=[]` dans les paramètres par défaut